pom.xml

```diff
@@ -1354,6 +1355,16 @@
     </reporting>
 
     <profiles>
+        <profile>
+            <id>fast</id>
+            <properties>
+                <maven.test.skip.exec>true</maven.test.skip.exec>
+                <maven.test.skip>true</maven.test.skip>
+            </properties>
+            <build>
+                <defaultGoal>clean install</defaultGoal>
```

Comment:
I really like default goals, but for the fast profile `clean` probably shouldn't be used, Maven is much better about incremental builds now a days

Reply:
ok I will remove clean goal.